test/integration/processes_poller.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class ProcessPollerTest < ActiveSupport::TestCase
+  def test_active_record_logger_nil
+    previous_logger = ActiveRecord::Base.logger
+    ActiveRecord::Base.logger = nil
+    pid = fork { SolidQueue::Supervisor.start }
+    sleep 1
+  ensure
+    ActiveRecord::Base.logger = previous_logger
+    terminate_process(pid) if pid && process_exists?(pid)
+  end
+
+  def test_active_record_logger_active_support_logger_dev_null
+    previous_logger = ActiveRecord::Base.logger
+    ActiveRecord::Base.logger = ActiveSupport::Logger.new("/dev/null")
+    pid = fork { SolidQueue::Supervisor.start }
+    sleep 1
+  ensure
+    ActiveRecord::Base.logger = previous_logger
+    terminate_process(pid) if pid && process_exists?(pid)
+  end
```

Comment:
Unrelated test coverage?

Reply:
Yeah I had a hard time trying to reproduce the bug, I don't know the library very well, so happy for pointers for better ways to do it